src/transport/SessionManager.h

```diff
@@ -210,8 +210,8 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate
     }
 
     // TODO: implements group sessions
-    Optional<SessionHandle> CreateGroupSession(GroupId group) { return mGroupSessions.AllocEntry(group, kUndefinedFabricIndex); }
-    Optional<SessionHandle> FindGroupSession(GroupId group) { return mGroupSessions.FindEntry(group, kUndefinedFabricIndex); }
+    Optional<SessionHandle> CreateGroupSession(GroupId group) { return mGroupSessions.AllocEntry(group, 1); }
+    Optional<SessionHandle> FindGroupSession(GroupId group) { return mGroupSessions.FindEntry(group, 1); }
```

Comment:
Why is a constant being replaced by a magic # here?  What does `1` mean?

Reply:
When the session refactor was merged the comment associated with the magic # was deleted. I cleaned up this portion of code and added the comment were the magic # is present.